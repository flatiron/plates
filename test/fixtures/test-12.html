<ul><li class="name"><span class="first"></span>:<span class="middle"></span><span class="last"></span></li></ul>
