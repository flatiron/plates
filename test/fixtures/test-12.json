{
  "name": [
    {
      "first"  : "John",
      "middle" : "C",
      "last"   : "Smith"
    },
    {
      "first"  : "Jane",
      "middle" : "A",
      "last"   : "Doe"
    }
  ]
}
