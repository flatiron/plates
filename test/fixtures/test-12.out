<ul><li class="name"><span class="first">John</span>:<span class="middle">C</span><span class="last">Smith</span></li><li class="name"><span class="first">Jane</span>:<span class="middle">A</span><span class="last">Doe</span></li></ul>
